src/main/java/frc/robot/RobotContainer.java

```diff
@@ -681,29 +681,29 @@ public Command scoreCoral() {
                 elevator.setHeight(ElevatorConstants.STOW_METER).withTimeout(0.0),
                 endEffector.stopMotor())
             .withTimeout(0.5);
-
+    // start here
     Command deAlgae =
         Commands.sequence(
                 endEffector.shoot(EndEffectorConstants.VOLTAGE_L4).onlyIf(IS_L4),
```

Comment:
Do we have extra overhead on this due to having 4 commands sequenced, that all get checked one by one?
Would it be faster to have one `endEffector.shoot` command with the voltage supplied at runtime, based on current target level?